conda-launchers/recipe.yaml

```diff
@@ -0,0 +1,153 @@
+context:
+  name: conda-launchers
+  version: "25.0.0"
+  build_number: 0
+
+recipe:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - url: https://raw.githubusercontent.com/python/cpython/3.7/PC/launcher.c
+    sha256: a3167e5908cefa942f63e0de5c6ec7a929e8ca82a382537ab8292e5aaa82ca2e
+    file_name: launcher.c.orig
+    patches:
+      - ../src/cpython-launcher-c-mods-for-setuptools.3.7.patch
+  - url: https://raw.githubusercontent.com/python/cpython/3.7/LICENSE
+    sha256: 96e4f59524cde5af4a2ea837ef5e52b65e51f1f999825fd8a9ec3b444cb82aea
+    file_name: cpython-LICENSE
+  - path: ../src
+
+build:
+  number: ${{ build_number }}
+  skip:
+    - unix
+    - win and arm64 # this ensures builds only trigger on win-64
+
+outputs:
+  - package:
+      name: ${{ name }}
+
+    build:
+      noarch: generic
+
+    requirements:
+      run:
+        - ${{ name }}_win-32 ${{ version }}
+        - ${{ name }}_win-64 ${{ version }}
+        - ${{ name }}_win-arm64 ${{ version }}
+
+    tests:
+      - requirements:
+          run:
+            - ${{ name }}_win-32 ${{ version }}
+            - ${{ name }}_win-64 ${{ version }}
+            - ${{ name }}_win-arm64 ${{ version }}
+        script:
+          # cli
+          - if not exist %PREFIX%\Scripts\cli-32.exe exit 1
+          - if not exist %PREFIX%\Scripts\cli-32-script.py exit 1
+          - call "%PREFIX%\Scripts\cli-32.exe"
+          - if not exist %PREFIX%\Scripts\cli-64.exe exit 1
+          - if not exist %PREFIX%\Scripts\cli-64-script.py exit 1
+          - call "%PREFIX%\Scripts\cli-64.exe"
+          - if not exist %PREFIX%\Scripts\cli-arm64.exe exit 1
+          - if not exist %PREFIX%\Scripts\cli-arm64-script.py exit 1
+          # gui
+          - if not exist %PREFIX%\Scripts\gui-32.exe exit 1
+          - if not exist %PREFIX%\Scripts\gui-32-script.pyw exit 1
+          - call "%PREFIX%\Scripts\gui-32.exe"
+          - if not exist %PREFIX%\Scripts\gui-64.exe exit 1
+          - if not exist %PREFIX%\Scripts\gui-64-script.pyw exit 1
+          - call "%PREFIX%\Scripts\gui-64.exe"
+          - if not exist %PREFIX%\Scripts\gui-arm64.exe exit 1
+          - if not exist %PREFIX%\Scripts\gui-arm64-script.pyw exit 1
```

Comment:
can't run arm64 executable on x64 vm, so no tests for arm64 launchers here.

Reply:
You could use platform selectors here if you want.